src/mesh/voxel_cloud.rs

```diff
@@ -128,111 +128,261 @@ impl VoxelCloud {
         voxel_cloud
     }
 
+    #[allow(dead_code)]
+    pub fn block_start(&self) -> &Point3<i32> {
+        &self.block_start
+    }
+
+    #[allow(dead_code)]
+    pub fn block_dimensions(&self) -> &Vector3<u32> {
+        &self.block_dimensions
+    }
+
+    #[allow(dead_code)]
+    pub fn voxel_dimensions(&self) -> &Vector3<f32> {
+        &self.voxel_dimensions
+    }
+
+    #[allow(dead_code)]
+    pub fn voxel_map_len(&self) -> usize {
+        self.voxel_map.len()
+    }
+
     /// For each existing voxel turn on all neighbor voxels to grow (offset) the
     /// volumes stored in the voxel cloud.
-    pub fn grow(&self) -> Self {
+    ///
+    /// FIXME: This is not the most efficient way of doing this, but this
+    /// function will become obsolete with Distance field.
```

Comment:
I agree, but let's let's put this FIXME in a normal comment, not a doc comment.

Reply:
#wasntme Autoformatter did this ;)

Reply:
bad autoformat, bad!